Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/Events/ReceiverBase.cs

```diff
@@ -16,8 +17,23 @@ public abstract class ReceiverBase
 
         public bool HideUnityEvents;
         protected UnityEvent uEvent;
+
+        /// <summary>
+        /// Each Receiver has a base Event it raises, (in addition to others).
+        /// </summary>
+        public UnityEvent Event
+        {
+            get { return uEvent; }
+            set { uEvent = value; }
+        }
+
         public MonoBehaviour Host;
 
+        public ReceiverBase()
+        {
+            uEvent = new UnityEvent();
```

Comment:
why not protected UnityEvent uEvent = new UnityEvent()? Reason for constructor?

Reply:
Removed this constructor.